src/themes/default/elements/icon.overrides

```diff
@@ -80,7 +86,7 @@ i.icon.arrow.right:before { content: "\f061"; }
 i.icon.arrow.up:before { content: "\f062"; }
 i.icon.arrows.alternate:before { content: "\f0b2"; }
 i.icon.arrows.alternate.horizontal:before { content: "\f337"; }
-i.icon.arrows.alternate.vertical:before { content: "\f338"; }
+i.icon.arrows.alternate.v:before { content: "\f338"; }
```

Comment:
This abbreviation `v` conflict the description in the header 
```
abbreviation are replaced by full words
i.icon.*-h         =>  i.icon.*.horizontal
i.icon.*-v         =>  i.icon.*.vertical
```

Reply:
I will take a look at this tomorrow, it will be a problem with [fomantic/create-fomantic-icons](https://github.com/fomantic/create-fomantic-icons)

Reply:
Opened a PR https://github.com/fomantic/create-fomantic-icons/pull/10